lib/types/beacon_chain/execution_payload_header.ex

```diff
@@ -48,13 +49,15 @@ defmodule Types.ExecutionPayloadHeader do
           fee_recipient: Types.execution_address(),
           state_root: Types.root(),
           receipts_root: Types.root(),
-          logs_bloom: binary(),
+          # size BYTES_PER_LOGS_BLOOM 256
+          logs_bloom: Types.bitvector(),
```

Comment:
Let's use BitVector.t here

Reply:
it looks like `logs_bloom` is a `vector` type according to the lighthouse implementation
`pub logs_bloom: FixedVector<u8, T::BytesPerLogsBloom>,`